### PR TITLE
Update the diagrams

### PR DESCRIPTION
**Pull request by martinthomson**
@MikeBishop, this should sort out #103 with Jana's comments.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -424,12 +426,11 @@ The fields in a Regular packet past the Common Header are the following:
...
   of 1.
 
-* AEAD Data: A Regular packet's Common Header and fields are authenticated,
-  but not encrypted.  The rest of a Regular packet, starting with the first
-  frame, is both authenticated and encrypted.  Immediately following the header,
-  Regular packets contain AEAD (Authenticated Encryption with Associated Data)
-  data.  After decryption, the plaintext consists of a sequence of
-  frames, as shown below (frames are described in {{frames}}).
+* AEAD Data: The remainder of a Regular packet is both authenticated and
+  encrypted once packet protection keys are available.  {{QUIC-TLS}} describes
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]: QUIC-TLS won't describe the structure of QUIC packets -- I don't think that needs to be referenced here.

Comment (martinthomson) [inline draft-ietf-quic-transport.md]: Hmm, you need a normative definition of how to protect/unprotect the plaintext and that is in QUIC-TLS.

Comment (MikeBishop) [inline draft-ietf-quic-transport.md]: Should it be?  TLS just provides the keys; QUIC does the actual protect/unprotect work.

Comment (janaiyengar) [inline draft-ietf-quic-transport.md]: Right, what Mike said. I suggest we leave this sentence out until warranted.

Comment (martinthomson) [inline draft-ietf-quic-transport.md]: Hmm, so QUIC-TLS actually does describe how to protect/unprotect packets.  We can move that text if you like and make that not-true, but you probably want to move the key schedule as well.  Then we don't have much left in the TLS doc.

**Pull request merged**